files/jquery.fullpage/update.json

```diff
@@ -0,0 +1,13 @@
+{
+  "packageManager": "github",
+  "name": "fullPage.js",
+  "repo": "alvarotrigo/fullPage.js",
+  "files": {
+    "basePath": "dist/",
+    "include": [
+      "jquery.fullpage.min.js",
+      "jquery.fullpage.min.css",
```

Comment:
There is no min.css here https://github.com/alvarotrigo/fullPage.js/tree/2.7.4
The bot will fail